algorithms/CPlusPlus/README.md

```diff
@@ -67,6 +67,7 @@
 6. [In order morris traversal](Trees/in-order-morris-traversal.cpp)
 7. [In order Predecessor and Successor](Trees/in-order-predecessor-and-successor.cpp)
 8. [Avl Tree](Trees/avl.cpp)
+9. [Min-Heap](Trees/min_heap.cpp)

```

Comment:
```suggestion
9. [Min-Heap](Trees/min-heap.cpp)
```
Please read the [readme.md](https://github.com/MakeContributions/DSA/blob/main/README.md) Contributing Specifications that you have to separate word with `-` not `_`

Reply:
My bad